examples/Example/src/bottomSheet/index.tsx

```diff
@@ -59,13 +81,17 @@ export class BottomSheet extends Component {
       extrapolate: 'clamp',
     });
   }
-  _onHeaderHandlerStateChange = ({ nativeEvent }) => {
+  _onHeaderHandlerStateChange = ({
```

Comment:
same here you can mark instance variable that is a method as private as well when it is not used anywhere else